js/App.js

```diff
@@ -12,7 +13,8 @@ export default class App extends Component {
   render() {
     return (
       <View style={styles.container}>
-        <Text>Welcome to AssistList</Text>
+        <LoginContainer />
+        {/* <Text>Welcome to AssistList</Text> */}
```

Comment:
Be careful. If you're not planning on using this commented out code, please remove it from your PR.